src/tests/test-aws-configuration.ts

```diff
@@ -56,8 +57,9 @@ if (awsUserDir && fs.existsSync(awsUserDir)) {
         }
       });
 
-      it("updates aws.config.region after each call", async () => {
-        for (const region of AWSRegions) {
+      it("updates aws.config.region after each call", async function () {
+        this.timeout(15000);
+        for (const region of AWSRegions.slice(0, 2)) {
```

Comment:
I'm a bit unclear why the tests need to work with a slice `AWSregions` instead of the whole set. Can you clarify?

Reply:
It was timing out with the full set due to the overhead with yubikeys, etc

Reply:
Also, a subset was sufficient for the purpose of the test.